libs/angular/src/lib/context-menu/context-menu.component.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  Output,
+  TemplateRef,
+  ViewChild,
+} from '@angular/core'
+import { DropdownOption } from '@sebgroup/extract'
+
+@Component({
+  selector: 'ngg-context-menu',
+  templateUrl: './context-menu.component.html',
+})
+export class NggContextMenuComponent {
+  @Input() direction: 'ltr' | 'rtl' = 'ltr'
```

Comment:
maybe could be opened to different directions(to top/left) if doesnt fit into screen.

Reply:
It could but would complicate the calculation and add edge cases, this can be developed with later version when someone needs this functionality